projects/plugins/boost/app/modules/optimizations/minify/Minify.php

```diff
@@ -24,49 +21,33 @@ public function setup() {
 			return;
 		}
 
-		add_action( Config::get_cron_cache_cleanup_hook(), 'jetpack_boost_page_optimize_cache_cleanup' );
-		register_deactivation_hook( JETPACK_BOOST_PATH, 'jetpack_boost_page_optimize_deactivate' );
-		register_uninstall_hook( JETPACK_BOOST_PATH, 'jetpack_boost_page_optimize_uninstall' );
+		if ( ! self::$called ) {
+			add_action( Config::get_cron_cache_cleanup_hook(), 'jetpack_boost_page_optimize_cache_cleanup' );
```

Comment:
Since this is used only once and it's a plain string in `Config::` class, I think it's much better to have this as a plain string here because it helps with code regex serachability later on.